packages/front-end/public/index.html

```diff
@@ -28,6 +28,8 @@
     <base href="/" />
   </head>
   <body>
+    <script src="/runtimeConfig.js" type="text/javascript"></script>
```

Comment:
Are you sure webpack will know there is a runtime config ? i see it got removed from the bundle.